### ISSUE TITLE
UI Breaks When Using `filament-peek` CSS in Custom Theme After Build

### ISSUE THREAD

**Issue by abbasmashaddy72** [pboivin/filament-peek#139]
**Problem Description:**

I was using **`@import "/vendor/pboivin/filament-peek/resources/css/plugin.css";`** in my custom CSS file, and everything worked fine. However, after running `npm run build`, the UI got messed up. Upon debugging, I found that the issue was related to this CSS file. 

When I commented out the `plugin.css` import, the issue was resolved, and the UI returned to normal. To temporarily fix the issue, I replaced the `plugin.css` import with the following:

```css
@import "/vendor/pboivin/filament-peek/resources/dist/filament-peek.css";
```

With this change, the UI worked correctly after the build.

### Relevant Files:

**`theme.css`** (before fix):
```css
/* Filament main theme CSS */
@import "/vendor/filament/filament/resources/css/theme.css";

/* Vendor CSS files */
@import "/vendor/awcodes/filament-table-repeater/resources/css/plugin.css";
@import "/vendor/awcodes/filament-tiptap-editor/resources/css/plugin.css";
@import "/vendor/pboivin/filament-peek/resources/dist/filament-peek.css";
/* @import "/vendor/pboivin/filament-peek/resources/css/plugin.css"; */
@import "/vendor/ysfkaya/filament-phone-input/dist/css/filament-phone-input.css";

/* Custom CSS files */
@import "./custom.css";
@import "./scroll-bar.css";
@import "./eclipse.css";

/* Tailwind configuration */
@config 'tailwind.config.js';
```

**`tailwind.conf.js`:**
```js
import preset from "../../../vendor/filament/filament/tailwind.config.preset";

export default {
    presets: [preset],
    content: [
        "./app/Filament/**/*.php",
        "./resources/views/filament/**/*.blade.php",
        "./Modules/**/*.php",
        "./vendor/awcodes/filament-table-repeater/resources/**/*.blade.php",
        "./vendor/awcodes/filament-tiptap-editor/resources/**/*.blade.php",
        "./vendor/awcodes/filament-quick-create/resources/**/*.blade.php",
        "./vendor/bezhansalleh/filament-language-switch/resources/views/language-switch.blade.php",
        "./vendor/bezhansalleh/filament-panel-switch/resources/views/panel-switch-menu.blade.php",
        "./vendor/filament/**/*.blade.php",
        "./vendor/pboivin/filament-peek/resources/**/*.blade.php",
        "./vendor/lara-zeus/**/*.blade.php",
    ],
    plugins: [require("tailwind-extended-shadows")],
};
```

**`package.json`:**
```json
{
    "private": true,
    "type": "module",
    "scripts": {
        "dev": "vite",
        "build": "vite build"
    },
    "devDependencies": {
        "@midzer/tobii": "^2.5.0",
        "@tailwindcss/forms": "^0.5.9",
        "@tailwindcss/typography": "^0.5.15",
        "autoprefixer": "^10.4.19",
        "axios": "^1.7.7",
        "flowbite": "^2.5.2",
        "laravel-vite-plugin": "^1.0",
        "postcss": "^8.4.47",
        "postcss-nesting": "^12.1.5",
        "remixicon": "^4.3.0",
        "sass": "^1.79.4",
        "tailwind-extended-shadows": "^0.4.0",
        "tailwindcss": "^3.4.13",
        "tiny-slider": "^2.9.4",
        "vite": "^5.4.8"
    }
}
```

### Screenshots:

1. **UI with Default Theme and Using `filament-peek` CSS (broken UI)**:
   ![image](https://github.com/user-attachments/assets/3278d09e-529f-46d4-8597-4ca63b010338)

2. **UI with Default Theme After Removing `filament-peek` CSS (issue resolved)**:
   ![image](https://github.com/user-attachments/assets/44eb4fe3-c66f-4cad-a643-c042acd54f4b)

3. **UI with Custom Theme and Using Original `filament-peek` CSS (broken UI)**:
   ![image](https://github.com/user-attachments/assets/25a43dc0-5304-430b-b59b-cc5363b78c6c)

4. **UI with Custom Theme and Using Dist Version of `filament-peek` CSS (temporary fix)**:
   ![image](https://github.com/user-attachments/assets/8beb4bdd-5ab6-4576-9d8d-f8d1fb49db1a)

**Comment by pboivin** [comment]
Hey @abbasmashaddy72, thanks for taking the time to investigate and report this. I should have a fix today.

**Comment by pboivin** [comment]
The fix is available in v2.2.10.

You can either switch back to `plugin.css` and rebuild (`npm run build`) or stay with your solution of using the vendor css file, it's a good solution.

**Comment by abbasmashaddy72** [comment]
Thanks For the Fix, I will check the updated versions.

Works Perfectly Bro.
